examples/whisper/prepare_whisper_configs.py

```diff
@@ -93,14 +120,24 @@ def main(raw_args=None):
         template_json["input_model"]["config"]["hf_config"]["from_pretrained_args"] = {"attn_implementation": "eager"}
 
     # set dataloader
-    metric_dataloader_kwargs = template_json["evaluators"]["common_evaluator"]["metrics"][0]["user_config"][
-        "func_kwargs"
-    ]["dataloader_func"]
-    metric_dataloader_kwargs["model_name"] = model_name
-    metric_dataloader_kwargs["use_audio_decoder"] = not args.no_audio_decoder
+    if not args.skip_evaluation:
+        metric_dataloader_kwargs = template_json["evaluators"]["common_evaluator"]["metrics"][0]["user_config"][
+            "func_kwargs"
+        ]["dataloader_func"]
+        metric_dataloader_kwargs["model_name"] = model_name
+        metric_dataloader_kwargs["use_audio_decoder"] = not args.no_audio_decoder
+    else:
+        del template_json["evaluators"]
+        template_json["engine"]["evaluator"] = None
 
     # update multi-lingual support
     template_json["passes"]["insert_beam_search"]["config"]["use_forced_decoder_ids"] = args.multilingual
+    # update predict timestep
+    template_json["passes"]["insert_beam_search"]["config"]["use_logits_processor"] = args.enable_timestamps
+    # update no audio decoder
+    template_json["passes"]["prepost"]["config"]["tool_command_args"]["use_audio_decoder"] = not args.no_audio_decoder
+    # update atol
+    template_json["passes"]["mixed_precision"]["config"]["atol"] = args.atol
```

Comment:
Any reason move this config setup here? For some pass flows we don't have mixed precision pass.

Reply:
We are updating the template config which has all passes. So, I moved all template related updates to the same place. In the for loop, we will only update the values related to `device` or `precision`.

Reply:
If so, should we also update the `Performs optimization pipeline:` part in README.md?

Reply:
For which part? The template json by itself doesn't work. We just use it to hold the pass config params and other settings.